src/shared/components/footer/style.css

```diff
@@ -86,3 +92,14 @@
 .inputSubmit:hover {
   background-color: green;
 }
+
+@media mediaSmall {
+  .address {
+    margin-bottom: 15px;
+  }
+
+  .links {
+    margin-bottom: 15px;
+    margin-left: -10px;
```

Comment:
The current site is also using negative margin for this, so going for parity for now.